torchvision/datasets/video_utils.py

```diff
@@ -0,0 +1,110 @@
+import bisect
+import torch
+from torchvision.io import read_video_timestamps, read_video
+
+
+def unfold(tensor, size, step, dilation):
+    """
```

Comment:
could you just confirm that I understand this correcly? Does this return the 2D tensor with num_rows=total_len/step such that row element has size*dilation length and the first element of each row is step apart?

Reply:
I'll improve the documentation.

This will return a 2d tensor, where the number of rows is the number of windows, and the columns is the number of elements in a window.

For example, for a `[1, 2, 3, 4, 5, 6]` tensor, with `size=3`, `step=1` and `dilation=2`, we will have
```
[[1, 3, 5],
 [2, 4, 6]]
```

Reply:
That's what I was thinking of - sounds good.